js/ui/multi_view.js

```diff
@@ -232,6 +232,19 @@ var MultiView = CollectionWidget.inherit({
         this.callBase();
     },
 
+    _updateIndicesAfterIndex: function(index) {
+        this.callBase(index);
+        this._deferredItems.splice(index + 1, 1);
+        for(var i = index + 1; i < this.option("items").length; i++) {
```

Comment:
The exception may be thrown if items are undefined.

Reply:
I have added conditions to avoid errors